images/kubernetes-event-exporter/tests/main.tf

```diff
@@ -0,0 +1,31 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_string" "ref" { input = var.digest }
```

Comment:
Great to have a helm chart for test coverage. Sometimes this may not always catch issues upon startup however. It would be great if we could expand this with a shell test to launch the container and check the startup logs for known good log entries.

Theres an example here with parts that could potentially be re-used: s[see here](https://github.com/chainguard-images/images/blob/main/images/keycloak/tests/keycloak-production-mode.sh).  And we can also see in the terraform how that test is invoked.

Fancy having a go at implementing similar here? it'd be much simpler than the one referenced - just defining some logs and checking on startup those logs exist.

Quick look at the containers runtime logs, these could be log lines to match:
```
Started container event-exporter
Started container event-exporter
```

Reply:
K8s event exporter requires to connect to the k8s cluster from inside the containers, so we need some workaround to make our k3d accessible to the containers. There is not much documentation available to refer to work on this, so since this image is an intermediate step in building the bitnami image, is it okay to keep to revisit this one.